packages/serverless/src/apollo/app/common.ts

```diff
@@ -35,6 +35,6 @@ export const indexName = Joi.string()
   .required()
   .pattern(/^[_a-z]+[_a-z0-9]+/i);
 
-export const indexNumber = Joi.number().required();
+export const indexNumber = Joi.number().unsafe().required();
```

Comment:
Incase of big number it should be encoded as a string and check with `/^[0-9]+/`.

Reply:
Fixed